### ISSUE TITLE
Is this project maintained?

### ISSUE THREAD

**Issue by jpulec** [platanus/angular-restmod#384]
Is this project still being maintained? Looks like there hasn't been a change since last October. I use this library fairly extensively at my workplace.

If the current maintainers no longer look after this project, is there any way I could be made a contributor or take over maintainership?

**Comment by ttonyh** [comment]
I agree, can we please get some clarification on this subject. As the OP said, the last change was in October 2015; I'd like to know what the future is for this project or if it needs a new maintainer.

**Comment by werner291** [comment]
Considering the lack of responses or activity, I think we can consider the project abandoned. Since this issue is quite high up, I guess you can just fork it and put a link in here.

**Comment by werner291** [comment]
Their test runner also seems to be failing.

**Comment by ttonyh** [comment]
Anyone interested in maintaining an official fork? I'd be happy to contribute.

**Comment by blackjid** [comment]
Hi guys, 

This library is core in all our projects, so is not in any way deprecated nor abandoned. We just haven't had any need for new features or bug fixing, and it's being working perfectly for our projects.

We just don't have the resources to use on developing new features, especially features we'll not use.

If someone is interested, and has the time or resources, to help maintaining or supporting the library as a collaborator, please contact us to see how we can better serve the community interests beside our own needs.

**Comment by jpulec** [comment]
Hi Juan.

Thanks for getting back to us. I'd be happy to spend some time trying to do some maintenance and support as a collaborator. You can reach me here through github. Let me know what next steps would be.

James

**Comment by jpulec** [comment]
Closing this, as I've been added as a collaborator.

**Comment by blackjid** [comment]
Well yes, big thanks to @jpulec who offer to help to maintain this project. 👏

**Comment by furqanaziz** [comment]
I see a lot many issues opened with this project and I sincerely can understand that you guys are not having enough time to provide support for this outstanding project. @blackjid is it possible you add me as collaborator. I am also working on adding style for JsonApi format and some add-on enhancements to this project and will add PR very soon.

**Comment by ttonyh** [comment]
Hi @jpulec, I see you were added as a collaborator. Can you please let us know if there's any plans for further maintenance? I don't seen any recent activity on here. Thank you.

Also, as a side note, if someone is willing to fork this repo, possibly give it a new name and a fresh new start, that would be awesome; I'd be happy to contribute. There are so many projects out there still using Angular 1 and there's still a lot of fixes, features and enhancements that we all could benefit from.

**Comment by jpulec** [comment]
@ttonyh I've stopped actively trying to work on this. We're currently looking to migrate off of angular entirely so it's not relevant to me anymore.

**Comment by ttonyh** [comment]
@jpulec Thanks for the quick response. Could you re-open this issue just so that it can get some more attention. As much as I'd love to start migrating our app to something else, that's not feasible right now and we rely on restmod heavily at the moment.

I think the best solution for the rest of us, is to fork this project and evolve it.
